python/cuml/test/test_dbscan.py

```diff
@@ -110,3 +143,4 @@ def test_dbscan_sklearn_comparison(name):
     assert(sk_n_clusters == cu_n_clusters)
 
     clusters_equal(sk_y_pred, cu_y_pred, sk_n_clusters)
+    cuml_dbscan.sync()
```

Comment:
Sync is needed before the call to `clusters_equal` no?

Reply:
Done